### PR TITLE
Fixes #22153: Various metadata.xml fixes

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/22153

### CODE REVIEW

File: policies/rudderc/tests/cases/general/min/min.xml [hunk 1]
```diff
@@ -25,9 +25,11 @@
   </AGENT>
   <SECTIONS>
     <SECTION name="Package present" id="d86ce2e5-d5b6-45cc-87e8-c11cca71d907" component="true" multivalued="true">
-      <VALUE id="d86ce2e5-d5b6-45cc-87e8-c11cca71d907">
-        <VALUE>htop</VALUE>
-      </VALUE>
+      <REPORTKEYS>
+        <VALUE id="d86ce2e5-d5b6-45cc-87e8-c11cca71d907">
+          htop
```
Comment (Fdall) [inline policies/rudderc/tests/cases/general/min/min.xml:30]: Are we sure that the xml reader does trim the whitespaces in the webapp?

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: policies/rudderc/tests/cases/general/min/min.xml (hunk 1 above)
Comment (amousset) [reply policies/rudderc/tests/cases/general/min/min.xml:30]: yes

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Pull request merged**